ntp-proto/src/algorithm/kalman/peer.rs

```diff
@@ -569,7 +746,7 @@ mod tests {
                 transmit_timestamp: Default::default(),
                 receive_timestamp: Default::default(),
                 localtime: base + NtpDuration::from_seconds(1000.0),
-                monotime: basei,
+                monotime: basei + std::time::Duration::from_secs(1000),
```

Comment:
why this change?

Reply:
Because we now check that both clocks progress at the same rate, otherwise the update is rejected.